src/Core/src/Graphics/Vector2.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+
+namespace Microsoft.Maui.Graphics
+{
+	public struct Vector2
```

Comment:
This is temporary, having Microsoft.Maui.Graphics we will use base types from there.